content/_index.md

```diff
@@ -48,7 +48,7 @@ end
 
 subgraph cloud[Cloud]
     snowplow[Snowplow BDP/OS Cloud]
-    snowflake[(Snowflake)]
+    warehouse[(Warehouse)]
```

Comment:
I feel like the streamlit part should have the arrow go _out_ of the warehouse? Also snowflake seems to be used elsewhere and the warehouse just sort of sat on the side by itself?